src/utils/Catalyst/index.ts

```diff
@@ -19,8 +19,8 @@ function getUsername(profile: CatalystProfile | null, address: string) {
 function getDclProfile(profile: CatalystProfile | null, address: string): DclProfile {
   const username = getUsername(profile, address)
   const hasAvatar = !!profile && !!profile.avatar
-  const avatar = hasAvatar ? profile.avatar.snapshots.face256 : DEFAULT_AVATAR_IMAGE
-  return { username, avatar, hasCustomAvatar: hasAvatar, address: address.toLowerCase() }
+  const avatarUrl = hasAvatar ? profile.avatar.snapshots.face256 : DEFAULT_AVATAR_IMAGE
```

Comment:
Why are we calling this `avatarUrl` if its an image? 🤔

Reply:
Because it is a url 🤔. And `avatar` from the decentraland endpoint, it is an object with a lot of data. So maybe `avatarImageUrl`?

Reply:
jajaja ok. Both are fine then! nvm